### PR TITLE
Fix false positives for interpolation in function-calc-no-invalid

### LINKED ISSUE #3908: Fix false positives for interpolation in function-calc-no-invalid
> Clearly describe the bug

False positive on `scss` interpolation

> Which rule, if any, is the bug related to?

e.g. `function-calc-no-invalid`

> What CSS is needed to reproduce the bug?

e.g.

```scss
a {
  width: calc(
    #{$min-value} +
    #{strip-unit($max-value - $min-value)} *
    (100vw - #{$min-vw})
  );
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "function-calc-no-invalid": true
  }
}
```

> Which version of stylelint are you using?

`9.10.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

SCSS

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
src/shared/styles/mixins/_fluid-type.scss
 4:42  ✖  Expected a valid expression   function-calc-no-invalid
```

### PR DESCRIPTION
**Pull request by ota-meshi**
> Which issue, if any, is this issue related to?

Closes #3908

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

File: lib/utils/parseCalcExpression/parser.jison [hunk 1]
```diff
@@ -50,6 +50,9 @@
 "("                                                   return 'LPAREN';
 ")"                                                   return 'RPAREN';
 
+#\{([\s\S]*?)\}                                       return 'UNKNOWN'; // scss variable
```
Comment (evilebottnawi) [inline lib/utils/parseCalcExpression/parser.jison]: // scss interpolation

Comment (ota-meshi) [reply lib/utils/parseCalcExpression/parser.jison]: Thank you for pointing out!
I fixed this.

**Comment by ota-meshi** [comment]
@hudochenkov Thank you for your review!
I added the following test case.

```css
      a {
        width: calc(
          #{$min-value} +
          #{strip-unit($max-value - $min-value)} *
          (100vw - #{$min-vw})
        );
      }
```

**Comment by hudochenkov** [comment]
-   Fixed: `function-calc-no-invalid` false positives for interpolation ([#4046](https://github.com/stylelint/stylelint/pull/4046)).

**Comment by Bigdragon13th** [comment]
When will this be released? Want to turn on the rule, but this issue prevent me.

**Comment by benjamincharity** [comment]
This issue is also a blocker for us 👍

**Comment by hudochenkov** [comment]
@Bigdragon13th @benjamincharity it will be released this week.

**Pull request merged**